src/app/profile/spaces/spaces.component.less

```diff
@@ -1,7 +1,7 @@
 @import (reference) '../../../assets/stylesheets/shared/osio.less';
 .spaces-page {
   position: relative;
-  margin-top: 20px;
+  margin-top: 40px;
```

Comment:
Set to `margin-top: 15px;` so that it matches the spacing of the other pages between navigation and content.